app/lib/ui/views/account/account_editing_screen.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:app/core/models/transporter.dart';
+import 'package:app/core/view_models/account_edit_view_model.dart';
+import 'package:app/ui/common/view_state.dart';
+import 'package:app/ui/widgets/utility/busy_overlay_screen.dart';
+import 'package:app/ui/widgets/utility/template_base_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_multi_formatter/flutter_multi_formatter.dart';
+
+class AccountEditingScreen extends StatefulWidget {
+  static const route = '/accountEditingScreen';
+  final formKey = GlobalKey<FormState>();
+  final Transporter account;
+
+  AccountEditingScreen({Key key, this.account}) : super(key: key);
```

Comment:
- [ ] Account should be required. No null accounts! This will change your null checks in the state below

_Your null checks below are also inconsistent. You check if the fields are null but then pass a possibly null account to the model. This is resolved by doing the above._